lib/dish.rb

```diff
@@ -0,0 +1,26 @@
+class Dish
+  attr_reader :name, :description, :price
+  def initialize(dish_hash = {name: "", description: "", price: 0})
```

Comment:
You can achieve this in a slightly more concise way
```
# this will still work even if you pass in a hash
def initialize(name: "", description: "", price: 0)
    @name = name
    @description = description
    @price = price
end
```

Reply:
Thanks that's good to know.